src/ui/public/saved_objects/saved_objects_client.js

```diff
@@ -126,10 +188,10 @@ export class SavedObjectsClient {
     return this._$http(options)
       .then(resp => get(resp, 'data'))
       .catch(resp => {
-        const respBody = resp.data || {};
+        const respBody = get(resp, 'data', {});
```

Comment:
It is not _identical_: with the `get` it's using the default value if `undefined`, while with the `||` it's doing it if `resp.data ` is falsy. As `resp.data` can't have any falsy values as "successful values" doing `||` is okey here. (I prefer `get`, though)